docfx.json

```diff
@@ -29,10 +29,22 @@
         "dest": "csharp/language-reference/language-specification",
         "exclude": [
           "README.md",
-          "LICENSE.md",
-          "**/includes/**"
+          "LICENSE.md"
         ]
       },
+      {
+        "files": [
+          "records.md",
```

Comment:
Same for this one as well. Not on TOC: https://review.docs.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/records?branch=pr-en-us-11026